src/mesh/tests/MeshTest.cpp

```diff
@@ -29,96 +29,6 @@ BOOST_AUTO_TEST_SUITE(MeshTests)
 
 BOOST_AUTO_TEST_SUITE(MeshTests)
 
-BOOST_AUTO_TEST_CASE(ComputeState_2D)
-{
-  PRECICE_TEST(1_rank);
-  mesh::Mesh mesh("MyMesh", 2, true, testing::nextMeshID());
-  // Create mesh
-  Vertex &v1 = mesh.createVertex(Vector2d(0.0, 0.0));
-  Vertex &v2 = mesh.createVertex(Vector2d(1.0, 0.0));
-  Vertex &v3 = mesh.createVertex(Vector2d(1.0, 1.0));
-  //
-  //
-  // *****
-  Edge &e1 = mesh.createEdge(v1, v2);
-  //     *
-  //     *  <---
-  // *****
-  Edge &e2 = mesh.createEdge(v2, v3);
-  mesh.computeState();
-
-  // Perform test validations
-  BOOST_TEST(equals(e1.getCenter(), Vector2d(0.5, 0.0)));
-  BOOST_TEST(equals(e2.getCenter(), Vector2d(1.0, 0.5)));
-  BOOST_TEST(e1.getEnclosingRadius() == 0.5);
-  BOOST_TEST(e2.getEnclosingRadius() == 0.5);
```

Comment:
I think we are losing all (unit) tests on these functions. Could you please add them to `EdgeTest` and `TriangleTest`.